app/src/main/java/app/revanced/integrations/patches/LithoFilterPatch.java

```diff
@@ -258,16 +258,11 @@ public GeneralBytecodeAdsPatch() {
     public boolean filter(final String path, final String identifier) {
         // Do not block on these
         if (ReVancedUtils.containsAny(path,
+                "comment_thread",
                 "home_video_with_context",
                 "related_video_with_context",
                 "search_video_with_context",
-                "download_",
-                "library_recent_shelf",
-                "menu",
-                "root",
-                "-count",
-                "-space",
-                "-button"
+                "horizontal_shelf"
```

Comment:
I remember couple ad components which are "horizontal_shelf". Do you consider that?

Reply:
This is the name used by history button components.

So, excluded it lead to the entire history button exclusion.

Reply:
> I remember couple ad components which are "horizontal_shelf". Do you consider that?

Anyway...you remember horizontal_video_shelf, that is another thing.